### PR TITLE
update docker init reference

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

Update docker init reference for 4.26.

### Related issues (optional)

DA-952

### CODE REVIEW

File: data/init-cli/docker_init.yaml [hunk 1]
```diff
@@ -197,6 +204,30 @@ examples: |-
...
+  CREATED: .dockerignore
+  CREATED: Dockerfile
+  CREATED: compose.yaml
+  CREATED: README.Docker.md
+  
+  ✔ Your Docker files are ready!
+  
+  Take a moment to review them and tailor them to your application.
+  
+  When you're ready, start your application by running: docker compose up --build
```
Comment (lizzthabet) [inline data/init-cli/docker_init.yaml:226]: there's one more piece of output text for the php template that you can include here:
```suggestion
  If your application requires specific PHP extensions, you can follow the instructions in the Dockerfile to add them.

  When you're ready, start your application by running: docker compose up --build
```
do you think it's also worth mentioning  on line 209 that any php extensions required by the app will need to be added manually to the dockerfile?

File: data/init-cli/docker_init.yaml [hunk 2]
```diff
@@ -197,6 +204,30 @@ examples: |-
...
+  
+  Your application will be available at http://localhost:9000
```
Comment (lizzthabet) [inline data/init-cli/docker_init.yaml:228]: all of the templates also have a new line related to the readme after this one:
```suggestion
  Your application will be available at http://localhost:9000
  
  Consult README.Docker.md for more information about using the generated files.
```

File: data/init-cli/docker_init.yaml [hunk 3]
```diff
@@ -65,6 +67,7 @@ examples: |-
     - .dockerignore
     - Dockerfile
     - compose.yaml
+    - README.Docker.md
   
   Let's get started!
```
Comment (lizzthabet) [inline data/init-cli/docker_init.yaml:72]: should php also be added to the list of templates below in the sample cli output on line 74?

**Review by lizzthabet — COMMENTED** [review]
this looks great! i just noted a few places where the php output could be more specific and the output has changed for with the new readme file.

**Comment by craig-osterhout** [comment]
Thanks @lizzthabet. It's been updated.

**Review by lizzthabet — APPROVED** [review]
lgtm! fyi, i did come across an unrelated [typo / formatting goof](https://github.com/docker/docs/pull/18795/files#diff-cb2ec74af7def8c687c4691e350efa77372182533fe7be3f40f95725c628d55bL18) on the init page, in case you want to fix it, but it isn't introduced on this pr.

Screenshot 2023-11-28 at 2 06 32 PM

**Review by dvdksn — APPROVED** [review]

**Pull request merged**